agent/util-scripts/pbench-metadata-log

```diff
@@ -153,12 +153,15 @@ function metadata_log_start {
     local ts=$3
     local log=$dir/metadata.log
 
+    name=$(basename $dir | sed 's/%/%%/')
+    config=$(echo $config | sed 's/%/%%/')
```

Comment:
Can we just create the entire `metadata.log` file as a temporary file and then use `sed 's/%/%%/g'` to process the entire file?

Note the trailing "`g`" on the `sed` is necessary.

Reply:
Ok, I will try it